### PR TITLE
NETBEANS-3020 Cleanup Welcome screen

### PR DESCRIPTION
**Pull request by geertjanw**
Remove the left panel in the 'What's New' tab and change 'Blogs' to 'News' in the right panel, so that the Apache NetBeans blogs provides the news in the Welcome Screen and so that the dependency on netbeans.org is removed.

### CODE REVIEW

File: nb/welcome/src/org/netbeans/modules/welcome/resources/Bundle.properties [hunk 1]
```diff
@@ -64,19 +60,19 @@ ACD_AllNews=Open the list of all news articles in the default web browser
 
 LBL_AllArticles=All Tutorials >>
 #NOI18N
-URL_AllArticles=http://netbeans.org/kb/kb.html?utm_source=netbeans&utm_campaign=welcomepage
+URL_AllArticles=https://netbeans.apache.org/help/index.html
```
Comment (jkovalsky) [inline nb/welcome/src/org/netbeans/modules/welcome/resources/Bundle.properties]: Maybe [https://netbeans.apache.org/kb/docs/index.html](https://netbeans.apache.org/kb/docs/index.html) here would be better?

**Comment by YTN01** [comment]
I saw few lines that got commented out. Any reason for not removing them?

**Comment by geertjanw** [comment]
> I saw few lines that got commented out. Any reason for not removing them?

True, fixing that.

File: nb/welcome/src/org/netbeans/modules/welcome/resources/Bundle.properties (hunk 1 above)
Comment (geertjanw) [reply nb/welcome/src/org/netbeans/modules/welcome/resources/Bundle.properties]: Problem is that the https://netbeans.apache.org/kb/docs/index.html page is incomplete at the moment and better not to expose it too much right now.

Comment (geertjanw) [reply nb/welcome/src/org/netbeans/modules/welcome/resources/Bundle.properties]: Problem is that https://netbeans.apache.org/kb/docs/index.html is incomplete right now and better not to highlight it at this point yet.

File: nb/welcome/src/org/netbeans/modules/welcome/resources/tutorial4.url [hunk 2]
```diff
@@ -1 +1 @@
-http://netbeans.org/kb/trails/php.html?utm_source=netbeans&utm_campaign=welcomepage
\ No newline at end of file
+https://netbeans.apache.org/kb/docs/php/index.html
```
Comment (vieiro) [inline nb/welcome/src/org/netbeans/modules/welcome/resources/tutorial4.url]: Hi @geertjanw , @neilcsmith-net , what about keeping the the "?utm_source=netbeans&utm_campaign=welcomepage" suffix even in the new URL? Who knows, maybe we can detect requests in the future...

**Comment by neilcsmith-net** [comment]
@vieiro aside from flexibility to redirect, the other reason for keeping links inbound from the IDE under `/nb/` was to allow for the option of tracking requests separately.

**Comment by vieiro** [comment]
@neilcsmith-net I know. Whatever is simpler for @geertjanw would do. Not sure we will be able to track anything, though (that's Infra business, IIRC).

**Comment by geertjanw** [comment]
I'm going to keep this PR as it is, if we need to do other things in this area, we can handle it via other PRs and issues. Thanks for the comments, squashing and merging now.

File: nb/welcome/src/org/netbeans/modules/welcome/resources/Bundle.properties [hunk 3]
```diff
@@ -97,7 +89,7 @@ URL_IncubatorLogo=http://incubator.apache.org
 #NOI18N
 URL_ApacheLogo=http://www.apache.org
```
Comment (thomas-mc-work) [inline nb/welcome/src/org/netbeans/modules/welcome/resources/Bundle.properties]: Why not `https://…`?

**Pull request merged**